### PR TITLE
Bump Ratatui to 0.28.0

### PR DESCRIPTION
**Pull request by orhun**
This PR fixes the compilation errors and warnings that came with Ratatui 0.28

closes #3

### CODE REVIEW

File: src/fx/glitch.rs [hunk 1]
```diff
@@ -128,7 +128,7 @@ impl Shader for Glitch {
             let x = cell.cell_idx % area.width as usize;
             let y = cell.cell_idx / area.width as usize;
             let pos = Position::new(area.x + x as u16, area.y + y as u16);
-            let c  = buf.get_mut(area.x + x as u16, area.y + y as u16);
+            let c  = buf.cell_mut(Position::new(area.x + x as u16, area.y + y as u16)).unwrap();
```
Comment (orhun) [inline src/fx/glitch.rs:131]: Let me know if you like the unwrap here or not :)

Comment (junkdog) [reply src/fx/glitch.rs:131]: yeah, it's fine - same as before. (even though i probably should handle it a bit more gracefully;))

**Review by junkdog — APPROVED** [review]
thanks! looking good :cookie:

nice api changes in ratatui btw

**Comment by junkdog** [comment]
i'm wrapping up the next release. i think i'll have it ready sometime tomorrow.

**Pull request merged**